omod/src/main/java/org/openmrs/module/reportingrest/web/resource/ReportRequestResource.java

```diff
@@ -170,6 +256,26 @@ else if (rep instanceof FullRepresentation) {
 			description.addProperty("renderCompleteDatetime");
 			description.addProperty("description");
 			description.addSelfLink();
+		} else if (rep instanceof RefRepresentation) {
+			description = new DelegatingResourceDescription();
+			description.addProperty("uuid");
+			//description.addProperty("baseCohort", Representation.DEFAULT);  TODO: Figure out how to support this
+			description.addProperty("parameterizable", "reportDefinition.parameterizable",
+					Representation.DEFAULT);
+			description.addProperty("parameterMappings", "reportDefinition.parameterMappings",
+					Representation.DEFAULT);

```

Comment:
Same here.  Can we fix this code so that the properties that are added to every representation are done just once, and then the conditional properties are added within the if statements?

Reply:
corrected to not duplicate adding properties